include/internal/libspdm_common_lib.h

```diff
@@ -342,6 +342,51 @@ typedef struct {
 } libspdm_msg_log_t;
 #endif /* LIBSPDM_ENABLE_MSG_LOG */
 
+#if LIBSPDM_FIPS_MODE
+typedef struct {
+    /**
+     * Tested algo flag: 0 represents that the algo is not tested.
+     * Bit0: tested HMAC_SHA256;
```

Comment:
Please remove bit enumeration here. 
Just say "See LIBSPDM_FIPS_SELF_TEST_xxx".

Reply:
Thanks. I have changed the comments.